Backend/Meta-TV2-api/Meta-TV2-AccessLayer/Controllers/ApiController.cs

```diff
@@ -43,7 +49,6 @@ public class Group : ControllerBase
         return add ? Ok() : BadRequest("Failed to add group.");
     }
 
-    [Authorize]

```

Comment:
GetGroups should probably require authorization. It could be a performance heavy call we do not want anyone to be able to invoke. And this endpoint might even be unnecessary, I dont think we will ever use it. My recommendation is to either require authorization or remove completely

Reply:
If this method is to exist, I believe it should not require authorization. It retrieves all active (non-archived) groups which is something a non-logged-in user should be able to do.

If this method is at all needed can be discussed. It is essentially doing the same thing as "GetGroupsByPage" but all in one call.